test/central_planner_test.py

```diff
@@ -343,19 +343,27 @@ def test_re_enable_failed_task_assistant(self):
         self.sch.ping(worker='X')  # worker still alive
         self.assertEqual('PENDING', self.sch.task_list('', '')['A']['status'])
 
-    def test_fail_job_from_dead_worker_with_live_assistant(self):
+    def test_assistant_doesnt_keep_alive_task(self):
         self.setTime(0)
         self.sch.add_task(worker='X', task_id='A')
         self.assertEqual('A', self.sch.get_work(worker='X')['task_id'])
-        self.sch.add_worker('Y', [('assistant', True)])
+        self.sch.add_worker('Y', {'assistant': True})
 
-        self.setTime(600)
+        self.setTime(self.sch._config.remove_delay + 1)
```

Comment:
You shouldn't access private variables like _config. This breaks on the RPC tests.